tests/test_fetcher.py

```diff
@@ -5,6 +5,9 @@
 
 from kafka.protocol.offset import OffsetResponse
 from aiokafka.record.legacy_records import LegacyRecordBatchBuilder
+from aiokafka.record.default_records import DefaultRecordBatchBuilder
+from aiokafka.record.memory_records import MemoryRecords
+from aiokafka.consumer.fetcher import PartitionRecords
```

Comment:
Duplicate import

Reply:
Fixed in 49c4be4

Reply:
Apologies for force-pushing, did not notice that you already started the review.